exporter/zipkin/options.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package zipkin
+
+import "github.com/openzipkin/zipkin-go/model"
+
+type Option interface {
```

Comment:
I think this is a lot of ceremony to accomplish this. I know @rakyll also dislikes the functional options pattern. What about just adding an exported field to the exporter and then users can do:
```go
e := zipkin.NewExporter(...)
e.RemoteEndpoint = "..."
```

Reply:
Unfortunately that makes for an awkward API -- why is localEndpoint an unexported field but RemoteEndpoint is exported? RemoteEndpoint being exported makes it seem then like the important field yet localEndpoint has been the seemingly most mandatory one.

Considering a case by case basis this is the need for an option because:
a) the exporter has been started and localEndpoint which is the more important field isn't exposed
b) for the 1+ year that this exporter has existed, no one has needed remoteEndpoint and am the first, hence the "optionality"